merkato/merkato.py

```diff
@@ -397,44 +524,63 @@ def modify_settings(self, settings):
         # replace old settings with new settings
         pass
 
+
     def add_reserve(self):
+        ''' TODO: Function comment
```

Comment:
does this need to be implemented for this feature?

Reply:
How do we initially set reserve again?

Reply:
in the merkato init, reserve is inputted by the user and then used to distribute ladders

Reply:
This isn't necessary (yet).